src/server/index.ts

```diff
@@ -1,14 +1,15 @@
 import * as dotenv from 'dotenv';
 import { Client, ClientUser } from 'discord.js';
 import { quoteDelete, quoteEvent } from './service';
+import { debugCommand } from './command/debugCommand';
 
 dotenv.config();
 const token = process.env.DISCORD_TOKEN;
-if (!token) {
-  console.error(
+export const prefix = process.env.PREFIX;
+if (!token || !prefix) {
```

Comment:
おそらく `PREFIX` の指定が無くても `!` とみなして動作するようなのでチェックは不要でしょう.

また, エントリポイントから何かを `export` するのは複雑化の原因ですので, こうせずに `debugCommand` 関数の引数を追加して `prefix` を受け取るようにしてみましょう.
```suggestion
const prefix = process.env.PREFIX || '!';
if (!token) {
```